Runtime/Tabs/TabsGroup.cs

```diff
@@ -9,7 +9,20 @@ internal class TabsGroup : ITabsGroup
     {
         /// <inheritdoc />
         [DataMember]
-        public int Selected { get; set; }
+        public int Selected
+        {
+            get
+            {
+                return selected;
+            }
+            
+            set
+            {
+                selected = value;
+            }
+        }
+
+        private int selected;
```

Comment:
Two options:
1. Don't think we need this anymore. Was introduced in a previous iteration where logic was within the getter and setter of the property but that's not the case anymore
2. Trigger OnSelected und OnUnselected in here which makes it less errorprone when actually using it -> I do not have to change the value and trigger the method

Reply:
Compare with TabsGroupDrawer change value (line 52)

Reply:
removed it